package/network/services/openvpn/files/openvpn.init

```diff
@@ -42,6 +42,7 @@ append_params() {
 		config_get v "$s" "$p"
 		IFS="$LIST_SEP"
 		for v in $v; do
+			[ "$v" == "frames_only" ] && [ "$p" == "compress" ] && unset v && append_param "$s" "$p" && echo >> "/var/etc/openvpn-$s.conf"
 			[ -n "$v" ] && [ "$p" != "push" ] && append_param "$s" "$p" && echo " $v" >> "/var/etc/openvpn-$s.conf"
 			[ -n "$v" ] && [ "$p" == "push" ] && append_param "$s" "$p" && echo " \"$v\"" >> "/var/etc/openvpn-$s.conf"
```

Comment:
So, this "==" should be replaced to.

Reply:
Yes, you may do this in a separate patch if you like.

Reply:
OK, I've added an additional commit.

Reply:
Thanks, please add a description to it, something that says why it's changed (i.e. it's bash syntax and this is /bin/sh ...)

Reply:
Done.